test/tf/linode/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "linode" {
+  version = "~> 1.5.0"
+}
+
+resource "linode_instance" "tagged" {
+  count              = 2
+  image              = "${var.linode_image}"
+  label              = "${var.prefix}-tagged-${count.index}"
```

Comment:
I'd add some randomness here. Some of the other providers use the random pet resource. This allows multiple tests to run safely in parallel.

Reply:
Just so happened the other providers I looked at did no have this, but I do see it now. I'll work on adding some randomness into the tests, similar to what I've found in the packet and azure providers.

Reply:
I've added a unique ID for each test run that is included in the instance labels, similar to what other providers are doing. The tests will not have access to the unique ID, though, which seems to be the case with other providers, as well.